src/Centreon/Infrastructure/Monitoring/ResourceRepositoryRDB.php

```diff
@@ -104,7 +104,7 @@ final class ResourceRepositoryRDB extends AbstractRepositoryDRB implements Resou
     private $serviceConcordances = [
         'h.name' => 'sh.name',
         'h.alias' => 'sh.alias',
-        'h.address' => 'sh.address',
```

Comment:
could you let the possibility to filter on `h.address`, to avoid breaking changes